include/pcp/filter/filter.hpp

```diff
@@ -0,0 +1,23 @@
+#ifndef PCP_FILTER_FILTER_HPP
+#define PCP_FILTER_FILTER_HPP
+
+/**
+ * @file
+ * @defgroup filter filter
+ * The filter module
+ */
+
+/**
```

Comment:
What is this group for? Haven't you already defined the filter module previously?